Examples/Lenses/Info.md

```diff
@@ -0,0 +1,33 @@
+### AutoLenses
+_Contributed by [@filip_zawada](http://twitter.com/filip_zawada)_
+
+What are Lenses? Great explanation by @mbrandonw
+
+This script assumes you follow swift naming convention, e.g. structs start with an upper letter.
+
+### [Stencil template](AutoLenses.stencil)
+
+### Available variable annotations:
+
+- `skipHashing` allows you to skip variable from being compared.
```

Comment:
I guess this does not belong here and there should be a note about `AutoLenses` protocol